CHANGELOG.md

```diff
@@ -50,6 +50,7 @@
 ### ⬆️ Improved
 
 ### ✅ Added
+Added support for file upload configuration that let's you specify what types of files and images you want to allow or block from being uploaded. [3280](https://github.com/GetStream/stream-chat-android/pull/3280)
```

Comment:
```suggestion
Added support for file upload configuration that lets you specify what types of files and images you want to allow or block from being uploaded. [3280](https://github.com/GetStream/stream-chat-android/pull/3280)
```

Let's it a contraction of "let us" i.e. "let us go" -> "let's go".
In this case you're using the third person singular of "to let" -> "lets".